cli/src/spend_utils.rs

```diff
@@ -215,6 +216,21 @@ where
                 },
             ))
         }
+        SpendAmount::Available => {
+            let mut lamports = if from_pubkey == fee_pubkey {
+                from_balance.saturating_sub(fee)
+            } else {
+                from_balance
+            };
+            lamports = lamports.saturating_sub(from_rent_exempt_minimum);
```

Comment:
Sorry, this isn't quite right. If there's no active stake, the rent-exempt reserve *is* available to be withdrawn. So the rent-exempt minimum only needs to be subtracted when `active_stake` is Some.
I think we need to do this on L133 above.

Reply:
We can either:
- 1: subtract at line 133 and delete the sub at line 225. (The case SpendAmount::Available will finally do the same logic as SpendAmount::All in resolve_spend_message function).
- 2: Or just reset from_rent_exempt_minimum to 0 when active_stake is None (L133) and keep the rest of the code as it is.
- What do you prefer?

Reply:
I prefer option 1